portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting comments. */
+@WebServlet("/delete-comment")
```

Comment:
We should have generic "comment" servlet. In that single servlet, we should have the create (doPost), get(doGet), update (doPatch) and delete (doDelete) functionality. 

Java servlets have "doDelete" it seems that you can use? https://tomcat.apache.org/tomcat-5.5-doc/servletapi/javax/servlet/http/HttpServlet.html#doDelete(javax.servlet.http.HttpServletRequest,%20javax.servlet.http.HttpServletResponse)

Reply:
Done